src/Tasks/CreateItem.cs

```diff
@@ -128,14 +134,55 @@ private List<ITaskItem> CreateOutputItems(Dictionary<string, string> metadataTab
             return outputItems;
         }
 
+        /// <summary>
+        /// Attempts to expand wildcards and logs warnings or errors for attempted drive enumeration.
+        /// </summary>
+        private bool TryExpandingWildcards(ITaskItem[] expand, bool includeAttribute)
+        {
+            const string CreateItemTask = "CreateItem";
+            string attributeType;
+            string fileSpec;
+
+            FileMatcher.SearchAction searchAction;
+
+            if (includeAttribute)
+            {
+                (Include, searchAction, fileSpec) = ExpandWildcards(expand);
+                attributeType = XMakeAttributes.include;
+            }
+            else
+            {
+                (Exclude, searchAction, fileSpec) = ExpandWildcards(expand);
+                attributeType = XMakeAttributes.exclude;
+            }
+
+            // Log potential drive enumeration glob anomalies when applicable.
```

Comment:
Why are these logged separately?

Reply:
EngineFileUtilities, CreateItem, and TrackedDependencies each call `GetFiles()`, so this case had to be treated separately from the switch statement in EngineFileUtilities. ProjectFileErrorUtilities.ThrowInvalidProject(...) is not accessible from within CreateItem, so I had to log an error using the same message resource. We're ignoring the TrackedDependencies case for now, since no logger is present.